Procfile

```diff
@@ -1 +1 @@
-worker: node app.js
+worker: node --max-old-space-size=460 app.js
```

Comment:
We do this with a higher number at URBN and in my experience has been common practice, so worth it imo 👍🏼